client/src/components/exercise/lab9/pages/Repairs/DateRepair.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-empty-pattern */
+import React from "react";
+import { PropTypes } from "prop-types";
+import useRepairDate from "../../hooks/useRepairDate";
+import Repair from "../../../../body/Repair/Repair";
+import { GAME_STATES, REPAIR } from "../../../../../constants/lab9";
+/**
+ * Date Repair is a component that is responsible for passing logic into the universal
+ * repair page component, what this allows us to do is call the re-useable repair component
+ * with custom logic pertaining to that repair section.
+ * @param {String} user contains user id for data state and logging user input
+ * @returns
+ */
+const DateRepair = ({ user = "" }) => {
+  const { data, functions } = useRepairDate(user);
+  const {} = data;
+  const {} = functions;
+  return (
+    <Repair
+      path={`${REPAIR}/${GAME_STATES.REPAIR_DATE_REPAIR}`}
+      CodeImplementation={() => {}}
```

Comment:
expected input?:
<>
<CodeLine>
<CodeLine>
<CodeLineInput>
<>?

Reply:
@Mster115 Yeah so that would be the place that would be ingesting the code line or the wrapper, I should make more defined docs to cover that.